.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @user
```

Comment:
This does not seem right, I want to be the code owner, not `user` (whoever that is).